cli/utilities/containers.py

```diff
@@ -148,16 +148,19 @@ def rmi(self, image):
         if self.execute(sudo=True, long_running=True, cmd=cmd):
             raise ContainerRegistryError(f"Failed to remove container image '{image}'")
 
-    def inspect(self, image, format=None):
+    def inspect(self, image, format=None, grep=None):
         """Inspect container image
 
         Args:
             image (str): Container image url
             format (str): String to format
+            grep (str): String to grep from output
         """
         cmd = f"{self.base_cmd} inspect {image}"
         if format:
             cmd += f" --format={format}"
+        if grep:
+            cmd += f" | grep -i {grep}"
```

Comment:
instead use `--filter` parameter. Refer - https://docs.podman.io/en/stable/markdown/podman-images.1.html

Reply:
My bad. 

Don't add grep here. Get the data through this lib and then parse it accordingly in the test.